CDP4SiteDirectory/Views/Dialogs/PersonDialog.xaml

```diff
@@ -230,7 +231,16 @@
                         <dxe:PasswordBoxEdit Name="PasswordConfirmation" Text="{Binding Path=PasswordConfirmation, Mode=TwoWay, ValidatesOnDataErrors=True, UpdateSourceTrigger=PropertyChanged}" />
                     </Grid>
                 </lc:LayoutItem>
-
+                <lc:LayoutItem Label=" " Visibility="{Binding PwdEditIsChecked, Converter={StaticResource BooleanToVisibilityConverter }, ConverterParameter=Invert }">
```

Comment:
You can remove the `Label` attribute all together.

Further, I'm not exactly super sure about the visibility logic here. To me this warning should be showing on two conditions being true:

- The Person is being created (when editing an existing Person there is no way clientside to know whether the password is set)
- The password is NOT being set so when this boolean is False (if I understand correctly)

Reply:
I agree. The thing we want to achieve is a reminder to the user that the password has not been set at creation time. On update we don't know the password in the UI as existing passwords are not read from the WebService.

Reply:
@alexatrhea I don't like how it looks without the label parameter:
![image](https://user-images.githubusercontent.com/112615243/189861382-084ff1d3-f829-4dbe-b667-83fb3b5581d8.png)
Somehow I prefer it to be aligned with the text box above. What do you think?

Reply:
ah in this case I understand. Disregard then.